### DISCUSSION TITLE
posts/why-divulgative-work/

### DISCUSSION THREAD

**Discussion by giscus** [michaelneuper/website#38] [General]
# posts/why-divulgative-work/

I have a personal interest in divulgative work, as I feel a very common issue in research is spreading too thin in branches of knowledge without paying attention what other branches are doing. The novelty of ideas and the research on them has merit, but what I truly appreciate are survey papers that let you have a thorough evaluation of what is going on. In the research world, where you often need to combine knowledge in very compartimentalized areas and jump into new areas these papers are invaluable.

http://localhost:1313/posts/why-divulgative-work/

**Comment by xarxaxdev** [comment]
this should be in localhost?
